app/src/main/java/fr/free/nrw/commons/utils/ImageUtils.java

```diff
@@ -213,14 +230,30 @@ else if (result == ImageUtils.IMAGE_BLURRY)
             errorMessage = context.getString(R.string.upload_image_problem_blurry);
         else if (result == ImageUtils.IMAGE_DUPLICATE)
             errorMessage = context.getString(R.string.upload_image_problem_duplicate);
+        else if (result == ImageUtils.IMAGE_GEOLOCATION_DIFFERENT)
```

Comment:
As pointed out by @misaochan lets not combine the error messages. I like @nicolas-raoul's suggestion of showing a list of error messages instead. That way the signature of this method would change to: 

```
getErrorMessageForResult(Context context, @Result List<Int> result) 
```

And instead of having `else if` we can have just a bunch of `if` statements which will keep appending to the `errorMessage`

Reply:
I agree:)